platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/InboundConnectionHandler.java

```diff
@@ -107,19 +99,6 @@ public void handle(final Socket clientSocket) {
             dis = new SerializableDataInputStream(sis);
             dos = new SerializableDataOutputStream(sos);
 
```

Comment:
The version check was removed here, but I don't see a replacement anywhere. Are we wanting to get rid of the version check entirely?

Reply:
yes, we're getting rid of it in both the inbound and outbound handlers as the version check is done elsewhere and the previous flag to do them here is always disabled.